clients/web/src/views/connect.tsx

```diff
@@ -134,6 +135,9 @@ export default function Connect() {
         port={connectionStore.port}
         retry={tryToConnect}
       />
+      <Show when={!DEV}>
```

Comment:
Why only in prod?

Reply:
But you can still proxy the dev host to another IP, so it would probably make more sense to use

```ts
<Show when={!isServer && (globalThis.location?.hostname !== 'localhost')}>
```

Reply:
I think setting up the frontend locally and proxying it and wanting to use safari will be an edge case. Then you can just comment out the modal. On top of that the modal will not stop you from using DevTools anyways. 

Plus I don't really want to think about what else Safari will consider localhost. Your exception would be missing `127.0.0.1` as well.

If it is fine with you, I would keep as is for now.